src/core/sim.py

```diff
@@ -32,7 +34,15 @@ def step(self) -> PropagatedOutput:
         current_event = self.event_queue.get()
         self.state_time, self.observed_state = current_event.evaluate(self.state_time)
         # TODO: Feed outputs of sensor models into FSW and return actuator's state as part of `PropagatedOutput`
-
+        # Current implementation uses Shared Memory to pass data
```

Comment:
This comment is a bit confusing. Pass what kind of data? I'd also leave out `Current implementation`. That's a bit redundant.